companies/exor.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "bitel",
+    "relevant-countries": [
+        "nl"
+    ],
+    "categories": [
+        "Holding company"
```

Comment:
Only the categories in the schema are allowed:

https://github.com/datenanfragen/data/blob/61957c19df76e8f3f281530d54987f0184b1c9e6/schema.json#L30